engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/GetAtomFieldsNode.java

```diff
@@ -15,14 +19,32 @@
     description = "Gets the fields of an unresolved atom.",
     autoRegister = false)
 public abstract class GetAtomFieldsNode extends Node {
+  private static final String LIMIT = "3";
+
   static GetAtomFieldsNode build() {
     return GetAtomFieldsNodeGen.create();
   }
 
   abstract EnsoObject execute(Atom atom);
 
+  @Specialization(
+      limit = LIMIT,
```

Comment:
I didn't know the default is three. Spelling it out explicitly make no harm. On contrary - there are two uses of the `LIMIT` constant in this file and they _must be the same_ - otherwise the code crashes on `UnsupportedSpecializationException`. That's the reason I introduced a constant - otherwise I'd just used plain `"3"` ;-)